docs/06_how-to-guides/40_multi-index/how-to-instantiate-a-multi-index-table.md

```diff
@@ -130,4 +130,4 @@ class [[eosio::contract]] multi_index_example : public contract {
 ```
 
 [[Info | Full example location]]
-| A full example project demonstrating the instantiation and usage of multi index table can be found [here](https://github.com/EOSIO/eosio.cdt/tree/master/examples/multi_index_example).
\ No newline at end of file
+| A full example project demonstrating the instantiation and usage of multi index table can be found [here](https://github.com/EOSIO/eosio.cdt/tree/master/examples/multi_index_example).
```

Comment:
nothing change here... however github is seeing a change... maybe a CRLF at the end of the line?...

Reply:
extra space